buildmaster/LHCB_Z_13TEV_DIMUON_2022/metadata.yaml

```diff
@@ -0,0 +1,104 @@
+setname: "LHCB_Z_13TEV_DIMUON_2022"
+
+nnpdf_metadata:
+  nnpdf31_process: "Z0PT"
+  experiment: "LHCB"
+
+arXiv:
+  url: "https://arxiv.org/abs/2112.07458"
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1990313"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1990313"
+  version: 2
+
+version: 1
+version_comment: "First implementation of the data set"
+
+implemented_observables:
+
+  # pT data
+  - observable: { description: "Differential cross-section of Z-->µµ as a function of Z-transverse momentum",
+                  label: '$\frac{d\sigma}{dp_T^Z}$',
+                  units: "[pb]"}
+    observable_name: PT
+    process_type: Z0PT
+    ndata: 14
+    tables: {stat: table_8, eff:  table_11, data: table_15, sys:  table_20}
+    kinematics:
+      variables:
+        pT: {description: "transverse momentum", label: "$p_T^Z$", units: "$GeV$"}
+        sqrt_s: {description: "center of mass energy", label: '$\sqrt{s}$', units: "$GeV$"}
+      file: kinematics_pT.yaml
+    data_central: data_pT.yaml
+    data_uncertainties:
+      - uncertainties_pT.yaml
+    kinematic_coverage: [pT, sqrt_s]
+    plotting:
+      kinematics_override: identity
+      x_scale: log
+      dataset_label: 'LHCB Z 13 TeV: $\frac{d\sigma}{dp_T^Z}$'
+      plot_x: pT
+      figure_by: 
+        - pT 
+    theory:
+      FK_tables: 
+        - - LHCB_Z_13TEV_DIMUON_2022_DIF_PT
+      operation: 'null' 
+
+  # y^Z data
+  - observable: { description: "Differential cross-section of Z-->µµ as a function of Z-rapidity",
+                  label: '$\frac{d\sigma}{dy^Z}$',
+                  units: "[pb]"}
+    observable_name: YZ
+    process_type: Z0PT
+    ndata: 18
+    tables: {data: table_14, sys:  table_19, stat: table_7, eff:  table_10}
+    kinematics:
+      variables:
+        yZ: {description: "boson rapidity", label: "$y^Z$", units: ""}
+        sqrt_s: {description: "center of mass energy", label: '$\sqrt{s}$', units: "$GeV$"}
+      file: kinematics_yZ.yaml
+    data_central: data_yZ.yaml
+    data_uncertainties: uncertainties_yZ.yaml
+    kinematic_coverage: [yZ, sqrt_s]
+    plotting:
+      kinematics_override: identity
+      x_scale: log
+      dataset_label: 'LHCB Z 13 TeV: $\frac{d\sigma}{dy^Z}$'
+      plot_x: yZ
+      figure_by: 
+        - yZ
+    theory:
+      FK_tables: 
+        - - LHCB_Z_13TEV_DIMUON_2022_DIF_PT
+      operation: 'null' 
+
+# pT-y^Z data
+  - observable: { description: "Double differential cross-section of Z-->µµ as a function of Z-transverse momentum and Z-rapidity",
+                  label: '$\frac{d^2\sigma}{d p_T^Z y^Z}$',
+                  units: "[pb]"}
+    observable_name: PT_Y
```

Comment:
You cannot use underscore in the variable name, if you want a double differential you can use `PT-Y`.